runtime/moonbase/src/lib.rs

```diff
@@ -197,18 +199,17 @@ pub fn native_version() -> NativeVersion {
 
 const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 pub const NORMAL_WEIGHT: Weight = MAXIMUM_BLOCK_WEIGHT.saturating_mul(3).saturating_div(4);
-// Here we assume Ethereum's base fee of 21000 gas and convert to weight, but we
-// subtract roughly the cost of a balance transfer from it (about 1/3 the cost)
-// and some cost to account for per-byte-fee.
-// TODO: we should use benchmarking's overhead feature to measure this
-pub const EXTRINSIC_BASE_WEIGHT: Weight = Weight::from_ref_time(10000 * WEIGHT_PER_GAS);
```

Comment:
Removing this constant means we have to use the `10_000` magic number elsewhere (e.g. in tests below). In addition, extracting this value out of `BlockWeights` isn't straightforward. I think we should leave it.

Reply:
added the `extrinsic_base_weight` function and moved it into `moonbeam_runtime_common` crate.